server/handlers.go

```diff
@@ -123,6 +122,10 @@ func (s *Server) discoveryHandler() (http.HandlerFunc, error) {
 		d.ResponseTypes = append(d.ResponseTypes, responseType)
 	}
 	sort.Strings(d.ResponseTypes)
+	for scope := range s.supportedScopes {
```

Comment:
Do you want to make sure at least the default scope exists? i.e. append user provided scopes to
`Scopes:      []string{"openid", "email", "groups", "profile", "offline_access"}`

Reply:
It means all scopes are allowed